packages/rolldown/src/log/logging.ts

```diff
@@ -6,6 +6,11 @@ export type LogLevelWithError = LogLevel | 'error'
 export type RollupLog = any
```

Comment:
Some types in this file are duplicated. The same types exist in `src/types/misc`.